Recent change breaks builds in Go 1.14

Even though `go.mod` specifies 1.14: https://github.com/silenceper/gowatch/blob/master/go.mod#L3

This breaks 1.14 builds: https://github.com/silenceper/gowatch/pull/34/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R98

The correct module for 1.14 is `ioutil`: https://pkg.go.dev/io/ioutil#WriteFile

The resulting error is:
```
# github.com/silenceper/gowatch
--
  | src/github.com/silenceper/gowatch/main.go:98:8: undefined: os.WriteFile
```

--- a/main.go
+++ b/main.go
@@ -3,6 +3,7 @@ package main
 import (
 	"flag"
 	"fmt"
+	"io/ioutil"
 	"os"
 	path "path/filepath"
 	"runtime"
@@ -95,7 +96,7 @@ func main() {
 	// init gowatch.yml
 	if len(os.Args) > 1 && os.Args[1] == "init" {
 		if _, err := os.Stat("gowatch.yml"); os.IsNotExist(err) {
-			_ = os.WriteFile("gowatch.yml", []byte(defaultYml), 0755)
+			_ = ioutil.WriteFile("gowatch.yml", []byte(defaultYml), 0755)
 			fmt.Println("gowatch.yml file created to the current directory with the default settings")
 		} else {
 			fmt.Println("gowatch.yml has been exists")

